src/pages/en/guides/cms/storyblok.md

```diff
@@ -424,6 +439,9 @@ try {
     <title>Storyblok & Astro</title>
   </head>
   <body>
+    {story.full_slug === 'home' && (
```

Comment:
Referencing my previous comment. I think we can replace the content variable with a story variable. So the `<StoryblokComponent blok={story.content} />` will match in both examples and the reader have access to the story metadata.